### PR TITLE
Using the LAF for components colors

### PR DESCRIPTION
**Pull request by tokazio**

### CODE REVIEW

File: Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java [hunk 1]
```diff
@@ -367,10 +353,10 @@ private CalendarPanel(
     // Set the calendar panel to be opaque.
     setOpaque(true);
     // Shrink the buttons for previous and next year and month.
-    buttonPreviousYear.setMargin(new Insets(1, 2, 1, 2));
-    buttonNextYear.setMargin(new Insets(1, 2, 1, 2));
-    buttonPreviousMonth.setMargin(new Insets(1, 2, 1, 2));
-    buttonNextMonth.setMargin(new Insets(1, 2, 1, 2));
+   // buttonPreviousYear.setMargin(new Insets(1, 2, 1, 2));
```
Comment (tokazio) [inline Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java:356]: These margins are redefined later

**Comment by tokazio** [comment]
Sample code i've change to use the LAF colors (using flat laf in dark in my case).
Code should be merge with actual ability to parameter the different colors.

File: Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java [hunk 2]
```diff
@@ -36,16 +36,8 @@
...
-import java.awt.Dimension;
-import java.awt.Font;
-import java.awt.FontMetrics;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
-import java.awt.Point;
-import java.awt.Rectangle;
+
+import java.awt.*;
```
Comment (dbmalkovsky) [inline Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java:40]: Please follow the project standards for imports.  Wildcard or package level imports are not supported; each class use should be explicitly enumerated.  Thanks

**Comment by BlakeTNC** [comment]
closing for inactivity

**Pull request closed without merging**